src/components/StaffCollection/StaffCollection.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+//do misc import statements here from other components
+import StaffCard from '../StaffCard';
+
+import "./StaffCollectionscss";
+
+export default class StaffCollection extends React.Component {
+  render() {
+
+    const members = this.props.members;
+
+    const arrayStyles = {
```

Comment:
I know you copied this from my code but I made a mistake writing it in the first place lol. Let's try to only do styles via SCSS, not JS.